lib/ui/screens/coffee_selection.dart

```diff
@@ -3,8 +3,10 @@ import 'package:flupresso/model/services/state/coffee_service.dart';
 import 'package:flupresso/model/services/state/machine_service.dart';
 import 'package:flupresso/service_locator.dart';
 import 'package:flutter/material.dart';
-import 'package:flupresso/ui/theme.dart' as theme;
 import 'package:flutter_typeahead/flutter_typeahead.dart';
+import 'package:provider/provider.dart';
+
+import '../theme.dart';
```

Comment:
If we need to import it directly anyways, we can use it out of that file directly. There is no added benefit from using the provider, or am I missing something? A change in the theme will issue a full redraw of the app anyways, so we can use a statefull widget on a high node and use it to redraw the tree without the need of introducing more variables to be passed around.